apps/server/src/app/export.controller.ts

```diff
@@ -182,6 +184,40 @@ export class ExportController extends BaseController {
     }
   }
 
+  /**
+   * The publisherVersion() returns an array of publisher versions from the sonatype.org endpoint defined below.
+   */
+  @Get(':publisher-version')
+  public publisherVersion(): Promise<any> {
```

Comment:
Consider renaming to getPublisherVersions()

Reply:
fixed